plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -8,3 +8,7 @@ deployStandardMenuLabel=Deploy to App Engine Standard...
 deployFlexCommandDescription=Uploads the project to Google App Engine flexible environment.
 deployFlexCommandName=Deploy to App Engine Flexible
 deployFlexMenuLabel=Deploy to App Engine Flexible...
+
+deployFlexExistingWarJarCommandDescription=Uploads existing WAR/JAR to Google App Engine \
+ flexible environment.
+deployFlexExistingWarJarCommandName=Deploy Existing WAR/JAR to App Engine Flexible
```

Comment:
and this is the menu item text, right? delete "Existing", perhaps change to "Deploy Existing WAR/JAR file App Engine Flexible"

Reply:
Your last suggestion reads weird to me. ("... file App Engine Flexible"). You mean "Deploy Existing App Engine Flexible WAR/JAR File"?